rdm/0004-browser-support.md

```diff
@@ -0,0 +1,77 @@
+- Start Date: 2019-10-02
+- RFC PR: [5](https://github.com/inveniosoftware/rfcs/pull/5)
+- Authors: Guillaume Viger
+
+# Support Evergreen Browsers
+
+## Summary
+
+InvenioRDM commits to supporting secure, evergreen browsers that are themselves
+supported by their vendors.
+
+
+## Motivation
+
+As an InvenioRDM core or extension/plugin developer, I want to take advantage
```

Comment:
Consistency OCD, the template has this use cases as enumerations :)

Reply:
:heavy_check_mark: